### PR TITLE
SRS: Imported changes from #218

### PR DESCRIPTION
**Pull request by jGleitz**
… as far as they were applicable.

[PDF](http://jgleitz.github.io/Beagle/branches/2nd-screnario/Requirements%20Specification.pdf)

### CODE REVIEW

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Models.lyx [hunk 1]
```diff
@@ -132,18 +132,43 @@ All characters and organisations appearing in this work are fictitious.
...
+\begin_layout Plain Layout
+
+
+\backslash
+glsuserv{system}
+\end_layout
+
+\end_inset
+
+ is based on an early design that has grown overthe years with missing documenta
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: Typo

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Models.lyx]: Fixed, thank you

**Comment by RomanLangrehr** [comment]
Lgtm.

**Pull request merged**